### DISCUSSION TITLE
Get Control Center from Windows 11

### DISCUSSION THREAD

**Discussion by lightmg** [valinet/ExplorerPatcher#205] [Q&A] [answered]
Is it possible to display new Control Center from Windows 11 with Win10 taskbar enabled?

I found how it can be enabled (Properties -> turn off option "Hide Control Center button"), but now it's displayed just like this:
![image](https://user-images.githubusercontent.com/56725483/138761296-43ed5e01-d6bc-4b47-be17-cd3611408bab.png)
Instead of 
![image](https://user-images.githubusercontent.com/56725483/138761550-effaeecd-31ae-4099-9ea5-380660ecd48b.png)

**Comment by valinet** [comment, marked as answer]
That’s the way it shows in the Windows 10 taskbar. That’s simply how it was implemented by Microsoft. In the new taskbar, they custom draw it differently.

As an alternative, use the classic discrete selectors for network, sound and battery.

Thanks

**Reply by S1r3** [reply]
Sorry to necropost but what do you mean by "use the classic discrete selectors for network, sound and battery" I tried finding out what this is but have no clue what you mean.

**Reply by chip33** [reply]
On windows 11 the  battery, network and sound icon are combined as one. 
In windows 10 they are separate , discrete icons with individual flyouts.

Win 11:
![UnlistedPower_0-1630899259188](https://user-images.githubusercontent.com/48499740/152246655-1474b8ee-d470-4973-958a-6b3407ec89ea.png)

Win 10:  (I don't have a battery icon)
![Screenshot 2022-02-06 161007](https://user-images.githubusercontent.com/48499740/152690262-9068a700-da9d-4475-b9ce-1bdbee0bcdcc.png)
